airbyte-server/src/test/java/io/airbyte/server/handlers/SourceDefinitionsHandlerTest.java

```diff
@@ -324,54 +324,6 @@ void testGetDefinitionWithGrantForWorkspace() throws JsonValidationException, Co
     assertEquals(expectedSourceDefinitionRead, actualSourceDefinitionRead);
   }
 
-  @Test
```

Comment:
similar question here, is this change sneaking in from your other PR?

Reply:
ditto